Libs/vtkSegmentationCore/vtkBinaryLabelmapToClosedSurfaceConversionRule.cxx

```diff
@@ -95,39 +98,39 @@ unsigned int vtkBinaryLabelmapToClosedSurfaceConversionRule::GetConversionCost(
 //----------------------------------------------------------------------------
 vtkDataObject* vtkBinaryLabelmapToClosedSurfaceConversionRule::ConstructRepresentationObjectByRepresentation(std::string representationName)
 {
-  if ( !representationName.compare(this->GetSourceRepresentationName()) )
-    {
+  if (!representationName.compare(this->GetSourceRepresentationName()))
+  {
```

Comment:
* large style changes are generally done in a dedicated pull request
* in this context the existing style is consistent through the code base and with the old VTK style [^1] (as of v6.3.0[^2]), this means we should not change it.

Keep:

```
if (!representationName.compare(this->GetSourceRepresentationName()))
  {
  }
```

[^1]: https://slicer.readthedocs.io/en/latest/developer_guide/style_guide.html#c
[^2]: https://github.com/Kitware/VTK/blob/v6.3.0/Common/Core/vtkInformation.cxx

Reply:
In the meantime, review may be streamlined using this link. See https://github.com/Slicer/Slicer/pull/7224/files?w=1